actions/class.Tests.php

```diff
@@ -184,17 +186,30 @@ public function delete()
 
         if ($instance->isClass()) {
             $class = $this->getClass($instance->getUri());
-            $success = $this->getClassService()->deleteClass($class);
+
+            $classDeleter = $this->getClassDeleter();
+
+            try {
+                $classDeleter->delete($class);
```

Comment:
Does that mean that for other resources (assets, groups, test-takers, etc) we will need to do that?

Reply:
Nope 🙂 
I checked all necessary places, most of them use `delete` method from RDF controller